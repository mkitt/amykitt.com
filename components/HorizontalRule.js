// @flow
import React from 'react'
import css, { media3 } from '../styles/css'

const style = css(
  {
    maxWidth: 1440,
    marginRight: 'auto',
    marginLeft: 'auto',
    paddingRight: 20,
    paddingLeft: 20,
  },
  media3({
    paddingRight: 80,
    paddingLeft: 80,
  }),
)

const ruleStyle = css({
  display: 'block',
  height: 1,
  border: 0,
  backgroundColor: '#858585',
  margin: '1em 0',
  padding: 0,
})

export default (props: any) => (
  <div className={style} {...props}>
    <hr className={ruleStyle} />
  </div>
)
