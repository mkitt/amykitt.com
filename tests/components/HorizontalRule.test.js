import React from 'react'
import renderer from 'react-test-renderer'
import HorizontalRule from '../../components/HorizontalRule'

describe('HorizontalRule', () => {
  let component

  beforeEach(() => {
    component = (
      <HorizontalRule />
    )
  })

  it('shows the correct snapshot for the HorizontalRule component tree', () => {
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
