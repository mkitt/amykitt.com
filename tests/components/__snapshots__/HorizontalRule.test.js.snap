// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HorizontalRule shows the correct snapshot for the HorizontalRule component tree 1`] = `
<div
  className={
    Object {
      "data-css-1atcpp3": "",
    }
  }
>
  <hr
    className={
      Object {
        "data-css-1ebtlzk": "",
      }
    }
  />
</div>
`;
